packages/components/src/font-size-picker/index.js

```diff
@@ -92,6 +92,10 @@ function FontSizePicker(
 
 	const selectedFontSizeSlug = getSelectValueFromFontSize( fontSizes, value );
 
+	const currentFontSizeSR = `Currently selected font size: ${
```

Comment:
This probably need to be translated properly?

Reply:
I think it should be good now.